app/policies/submission_policy.rb

```diff
@@ -30,7 +30,7 @@ def view_files?
   end
 
   def html_content?
-    Rails.application.config.nbconvert_enabled && check?(:view_files?)
+    check?(:view_files?)
```

Comment:
because of the simplification here, you can delete the method definition altogether and instead specify an *alias* (see the top of this file)